docs/vnext/openapi.yaml

```diff
@@ -26,6 +26,17 @@ paths:
       summary: Create a resource reference
       description: |
         Generate a reference to a resource comprising a reference ID, regardless the existence of the resource.
+        
+        The reference ID MUST adhere to the format {flags}-{uniqueId}.
+        
+        ## {flags}
+        This section consists of two binary digits indicating the type of reference ID. The currently accepted values are `00` and `01`:
+        
+        - `00`: The reference ID is self-contained and requires no additional information for GET and DELETE operations.
+        - `01`: To perform GET and DELETE operations, the reference ID, along with the user-provided provider configuration, is necessary (e.g., the configuration includes a connection string).

```

Comment:
>01 [](http://example.com/codeflow?start=11&length=2)

Will the extensibility host enforce this convention?

Reply:
No, but the deployment engine will enforce it.